spec/index.adoc

```diff
@@ -4075,19 +4076,6 @@ S with
     system_time = T1
 ....
 
-Finally, the canister version can be incremented arbitrarily:
-
-Conditions::
-....
-    N0 = S.canister_version[CanisterId]
-    N1 > N0
-....
-State after::
-....
-S with
-    canister_version[CanisterId] = N1
-....
-
```

Comment:
I'd suggest to leave it in, unless you really want people to rely on the one-increment-per-message feature. It gives the replica some leeway.

Reply:
Reverted the original change.